precompiles/dapps-staking/src/lib.rs

```diff
@@ -253,6 +253,7 @@ where
             [0x77, 0xa0, 0xfe, 0x02] => Self::withdraw_unbonded()?,
             [0xc1, 0x3f, 0x4a, 0xf7] => Self::claim(input)?,
             _ => {
+                println!("selector{:x?}", selector);
```

Comment:
How does compiler work here? :scream:

Reply:
thanks for finding this!!

Reply:
Sure, but do you know why git action passed here?
Does it work locally for you?